delivery_carrier_label_postlogistics/postlogistics/web_service.py

```diff
@@ -28,6 +28,15 @@
 from openerp.tools.translate import _
 
 _compile_itemid = re.compile(r'[^0-9A-Za-z+\-_]')
+_logger = logging.getLogger(__name__)
+
+try:
```

Comment:
As it's declared on external_dependencies, you should make this import on `init_connection` and the methods where they are needed, and let the module manager to avoid the installation of the module in case the library is missing. This way, you avoid the annoying warning each time the server with this module uninstalled is started.